DataParser/dataParser.py

```diff
@@ -0,0 +1,138 @@
+import os
+import json
+import xmltodict
+import zipfile
+import pandas as pd
+from pathlib import Path
+import pytest
+import sys
+import filecmp
+
+
+class ooxml_file:
```

Comment:
Class names should use the CapWords convention - https://www.python.org/dev/peps/pep-0008/#class-names